src/algorithms/MPolyFactor.jl

```diff
@@ -1314,7 +1319,12 @@ function mfactor_squarefree_char_zero(a::E) where E
   # start with a monic version of a
   lc = coeff(a, 1)
   res.unit = R(lc)
-  res.fac = Dict{E, Int}(1//lc * a => 1)
+  if !is_unit(lc)
+    error("leading coefficient must be invertible")
+  end
+  # thus lc is invertible in K, which means that it is squarefree
+  # don't use `inv`, it is wrong for `BigInt`
+  res.fac = Dict{E, Int}(AbstractAlgebra.inv(lc) * a => 1)
```

Comment:
I don't understand, the line right above says bot to use `inv`?

maybe
```suggestion
  res.fac = Dict{E, Int}(a / lc => 1)
```

Reply:
`inv !== AbstractAlgebra.inv`. And `/` is illegal, since it is the wrong thing for `BigInt`.

Reply:
@fingolfin hopefully that answers your questions